assets/css/less/front-end/components/submit-listing/plan-selection-section.less

```diff
@@ -0,0 +1,77 @@
+/**
+ * Submit Listing.
+ *
+ * Plan Selection Section.
+ */
+
+.wpbdp-plan-selection-with-tip {
+	margin-top: 24px;
+}
+
+.wpbdp-plan-selection-list {
+	display: flex;
+	gap: 25px;
+	justify-content: center;
+
+	.wpbdp-plan {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		position: relative;
+		border: 1px solid @border-main;
+		border-radius: @border-radius-main;
+		padding: 22px;
+		text-align: center;
+
+		& > * {
+			flex-basis: 100%;
+		}
+	}
+
+	.wpbdp-plan-label {
+		color: @color-text;
+		font-size: 17px;
+		font-weight: 500;
+		line-height: @line-height-body;
+	}
+
+	.wpbdp-plan-price-amount {
+		color: @color-text;
+		font-size: @font-size-h2;
+		font-weight: 700;
+		line-height: @line-height-h2;
+	}
+
+	.wpbdp-plan-feature-list {
+		margin-top: 25px;
+		color: @color-text;
+		font-size: @font-size-body;
+		font-weight: 400;
+		line-height: @line-height-body;
+	}
+
+	.wpbdp-plan-action {
+		margin-top: 23px;
+		align-self: flex-end;
+		input[type="radio"] {
+			display: none;
+		}
+
+		.wpbdp-button {
+			width: 100%;
+		}
+	}
+
+	.wpbdp-msg {
+		display: flex;
+		display: none;
```

Comment:
I'm not sure why this is hidden. Here's what I saw before:
<img width="1234" alt="Screenshot 2023-08-10 at 6 18 44 PM" src="https://github.com/Strategy11/business-directory-plugin/assets/1116876/4007c1a7-8026-4412-afca-ec2aa30629e1">

But now nothing happens when I click, with no indication as to why it's not working.